include/nbl/asset/format/EFormat.h

```diff
@@ -484,6 +484,16 @@ const core::vector3du32_SIMD getBlockDimensions()
     }
 }
 
+inline uint32_t getMaxChannelBitDepth(asset::E_FORMAT _fmt)
+{
+#include "nbl/asset/format/impl/EFormat_getMaxChannelBitDepth.h"
+}
+template<asset::E_FORMAT _fmt>
+constexpr uint32_t getMaxChannelBitDepth()
+{
+#include "nbl/asset/format/impl/EFormat_getMaxChannelBitDepth.h"
+}
```

Comment:
no, don't want such function.

Each channel's bit depth needs to be compared INDIVIDUALLY